spec/index.html

```diff
@@ -898,7 +875,7 @@ <h3>Algorithm</h3>
           <ol>
             <li id="ca-2-1">For each <a>blank node</a> that is a component of <var>Q</var>,
               add a reference to <var>Q</var> from the
-              <a data-cite="INFRA#map-entry">map entry</a> for the
+              [= map/entry | map entry =] for the
```

Comment:
For whatever reason, I've been reluctant to use the `[= ... =]` form, but I suppose I need to catch up with the times :).

In this case, the `data-cite="INFRA#map-entry"` does seem to resolve to the proper location (https://infra.spec.whatwg.org/#map-entry), and I'm not sure why, or how I would figure out that "map/entry" is the proper way to reference the term.

Reply:
Perhaps that can be inferred from the "#map-entry" fragment identifier, which implies a hierarchical relationship.

Reply:
To find how to refer to an existing term, https://respec.org/xref/?term= is the right tool; the syntax `map/entry` comes from the fact that the term `entry` is defined as a concept in the context of `map`.